terraform-beyond-layer0/example-3/modules/guestbook_service/Dockerrun.aws.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
> You can just use modules to group a set of resources or provide a layer of abstraction

Definitely - but `example-3/modules/guestbook_service` is not that.
This is the exact same trap we fell into when first starting w/ Terraform.
Your module here is mixing the "blueprint" config and the "real" config.